client/components/admin_dashboard/Map.jsx

```diff
@@ -3,8 +3,8 @@ import { GoogleMap, LoadScript } from "@react-google-maps/api";
 import { googleApiKey, appointmentsMapId } from "@/constants";
 import React, { useRef } from "react";
 
-export default function Map({ appointments }) {
-  if (!appointments.length) {
+export default function Map({ initAppointments }) {
```

Comment:
```suggestion
export default function Map({ appointments }) {
```
Not sure it makes to call this initAppointments when it's the only version of appointments this will see

Reply:
ah right, because appointments state in ReservationTable is already initialized with initAppointments.. I'll change it.